client/src/components/controls/FileNameTimeDisplay.vue

```diff
@@ -22,13 +24,21 @@ export default defineComponent({
     });
     return {
       display,
+      frame,
+      trueVideoFrame,
+      currentTime,
     };
   },
 });
 </script>
 
 <template>
   <span>
-    {{ display }}
+    <span>
+      {{ display }}
+    </span>
+    <span class="border-radius mr-1">frame {{ frame }}</span>
+    <span class="border-radius mr-1">real {{ trueVideoFrame.toFixed(3) }}</span>
+    <span class="border-radius">ts {{ currentTime.toFixed(6) }}</span>
```

Comment:
Is this staying in here?  Is it relevant to most users or is it going to confuse them.  I could understand hiding it under an advanced thing.  Also the currentTime posted here has an accuracy issue of the `OnePTSTick` offset which I think can be seen in the `toFixed(6)`.  I don't know if meant to be the real `frameTime` or the real requested time of the client.

Reply:
No, I'll be removing it.